eclass/acct-group.eclass

```diff
@@ -156,8 +156,32 @@ acct-group_src_install() {
 acct-group_pkg_preinst() {
 	debug-print-function ${FUNCNAME} "${@}"
 
-	enewgroup ${ACCT_GROUP_ENFORCE_ID:+-F} "${ACCT_GROUP_NAME}" \
-		"${_ACCT_GROUP_ID}"
+	if [[ ${EUID} -ne 0 ]]; then
+		einfo "Insufficient privileges to execute ${FUNCNAME[0]}"
+		return
+	fi
+
+	if egetent group "${ACCT_GROUP_NAME}" >/dev/null; then
+		elog "Group ${ACCT_GROUP_NAME} already exists"
+		return
+	fi
+
+	local opts=( --system )
+
+	if [[ -z ${ACCT_GROUP_ENFORCE_ID} ]]; then
```

Comment:
No, [[ -z ]] is correct here. From the groupadd man page:
```
       -f, --force
           This option causes the command to simply exit with success status
           if the specified group already exists. When used with -g, and the
           specified GID already exists, another (unique) GID is chosen (i.e.
           -g is turned off).
```
When ACCT_GROUP_ENFORCE_ID is blank, we want groupadd to pick another GID instead of failing if the GID already exists.

Reply:
I will note that `useradd` does not have an option equivalent to `groupadd --force`. In acct-user.eclass, I added a `egetent passwd "${_ACCT_USER_ID}"` call to check for an existing UID before adding `--uid "${_ACCT_USER_ID}`.

If you prefer for consistency, I could update acct-group.eclass to do a similar check for the GID before adding `--gid ${_ACCT_GROUP_ID}`. I would then drop the `--force` option.

Reply:
Yeah, please do that. I find this whole `--force` option confusing, and I'm going to guess it's not portable anyway.

Reply:
Done.